src/routes/getProduct.js

```diff
@@ -0,0 +1,6 @@
+import Storage from '../services/storage';
+
+export default function getProduct(req, res) {
+    let product = Storage.get('products', [{ name: 'id', value: req.params.id }]) || {};
```

Comment:
What if there is no such product?

Reply:
If will not be found product with id === req.params.id, will be responded with an empty array.

Reply:
I don't quite understand the question, but I storage `reviews` in a memory and in a JSON as an array:
https://github.com/pakhuta/siarhei.pakhuta.nodejs/blob/5ece536df72039f47b1f31463847e6749f877eeb/src/models/product.js#L5
https://github.com/pakhuta/siarhei.pakhuta.nodejs/blob/5ece536df72039f47b1f31463847e6749f877eeb/data/products.json#L5

Reply:
Better to respond with 404.